sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HiveClientSuite.scala

```diff
@@ -66,6 +65,19 @@ class HiveClientSuite(version: String)
     client
   }
 
+  private def attr(name: String): Attribute = {
+    name match {
+      case "ds" =>
+        AttributeReference("ds", IntegerType)()
```

Comment:
This is error-prone, we should look up the attribute by name from `spark.table("test")`